pyrdp/mitm/MCSMITM.py

```diff
@@ -143,7 +148,9 @@ def onConnectResponse(self, pdu: MCSConnectResponsePDU):
 
             for index in range(len(serverData.networkData.channels)):
                 channelID = serverData.networkData.channels[index]
-                self.state.channelMap[channelID] = self.state.channelDefinitions[index].name
+                name = self.state.channelDefinitions[index].name
+                self.log.info("%(channelName)s <---> Channel #%(channelId)d", {"channelName": name, "channelId": channelID})
```

Comment:
Why do we log the channel ID? We always refer to the channels by their name, couldn't we just print the whole list of established channels without their ID?

Reply:
The mcs mitm doesnt have access to the association between channel name and chanel id, so this is needed to make sense of the statcounter's mcs stats